### DISCUSSION TITLE
Error in Random Forest (R Studio)

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#400] [Q&A] [answered]
Saya mengalami kendala ketika ingint melakukan prediksi menggunakan algoritma Random Forest. 

Kode saya : 
sampel = sample(1:nrow(data_rf), 0.75*nrow(data_rf), replace=T)
training = data.frame(data_rf)[sampel, ]
testing = data.frame(data_rf)[-sampel, ]
modelRF <- randomForest(Attack.Category..custom.~., data=training)

Errornya : 
Error in randomForest.default(m, y, ...) : 
Can not handle categorical predictors with more than 53 categories.

Padahal categorynya kurang dari 53
<img width="366" alt="image" src="https://user-images.githubusercontent.com/106010284/196134425-42c91e96-fda7-45a3-9d2a-20dd248bb115.png">

**Comment by finesaaa** [comment, marked as answer]
Selamat pagi, pak Yohanes. Apakah script dan data yang digunakan masih sama seperti yang Bapak kirim kepada saya ya?
